src/canvas/hooks/useDrawableCanvas.ts

```diff
@@ -121,23 +119,20 @@ const useDrawableCanvas = ({
 
   useEffect(() => {
     const handleMouseMove = (e: MouseEvent | TouchEvent) =>
-      window.requestAnimationFrame(() =>
```

Comment:
requestAnimationFrame was causing desynchronization with renders. It should not be used here